code/modules/uplink/uplink_items/uplink_stealth.dm

```diff
@@ -117,7 +117,6 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 	item = /obj/item/suppressor
 	cost = 1
 	surplus = 10
-	exclude_modes = list(/datum/game_mode/nuclear/clown_ops)
```

Comment:
The toy pistol and smg are available to clown ops and can also be suppressed, this is pretty futile.